less/home-page.less

```diff
@@ -1,10 +1,18 @@
 // Home Content
 
-.home {
 
   .intro {
 
+    .wide-boi {
+      height: 100%;
+      width: 100%;
+      @media @phone {
+        width: 100%;
```

Comment:
If you set the width to 100% on the base, you won't need to reset it in a media query to the same value - it'll automatically be 100%